jenkinsfiles/dev

```diff
@@ -1,75 +1,164 @@
+#!/usr/bin/env groovy
+
 @Library('pipeline-library') _
 
 pipeline {
     agent {
         label 'ec2-jdk11'
     }
-
```

Comment:
Maybe this is from the IntelliJ formatting, but lets keep the blank lines between the different blocks - I think it makes the whole Jenkinsfile a bit more readable. This is applicable to a bunch of places throughout the dev and stable Jenkinsfiles.